pkg/summarizer/summary_test.go

```diff
@@ -1105,21 +1106,24 @@ func TestOverallStatus(t *testing.T) {
 	}
 }
 
-func TestStatusMessage(t *testing.T) {
+func makeShim(v ...interface{}) []interface{} {
```

Comment:
Huh, not sure what my opinion is on makeShim; my intuition says this should be an explicit struct instead?

@fejta, opinion on this specifically?

Reply:
Good point, because it is not being used generally I can just wrap with a struct.